releasenotes/releasenotes_v5.rst

```diff
@@ -10,18 +10,37 @@ Current Firely Server release notes (v5.x)
 
 .. _vonk_releasenotes_5_4_0:
 
-Release 5.4.0, Paste The Date, 2023
------------------------------------
+Release 5.4.0, October 26th, 2023
+---------------------------------
 
 Features
 ^^^^^^^^
+#. Firely Server can now interact asynchronously with a message broker (like RabbitMq or Azure Service Bus) using PubSub plugin. Resources can be ingested into Firely Server using a Firely Server-specific message. Additionally, all changes within the Firely Server database are broadcasted to the message bus. For more details, see :ref:`PubSub`.
+#. Improved handling of $lastN where multiple Observations result in a tie based on their respective effective times.
+#. $lastN now uses a case-sensitive comparison for text-only Observations.
+#. $lastN now includes multi-codes in its grouping.
+#. Added documentation regarding MongoDB security best-practices. See :ref:`configure_mongodb_security`.
+#. Added support for MongoDB as a Serilog sink. See :ref:`configure_log_sinks`.
+#. Compartment-wide searches can now be executed on a system level.
```

Comment:
We should add that the  "Vonk.Smart" plugin  and the "Vonk.Plugin.SoFv2" are now interchangeable, This allows custom plugins such as $everything to be used with the latter. Before this would not work.